### DISCUSSION TITLE
如何将图表标题改为加粗宋体

### DISCUSSION THREAD

**Discussion by skyzh** [sjtug/SJTUThesis#623] [提问] [answered]
《上海交通大学关于本科生毕业设计(论文)工作的指导意见》中规定“每幅插图应有图序和图题(五号宋体加黑)”，但模版目前使用的是楷体。

**Comment by skyzh** [comment, marked as answer]
可以参考 https://github.com/sjtug/SJTUThesis/issues/308 和 https://github.com/sjtug/SJTUThesis/issues/621#issuecomment-842485961 的解答。SJTUThesis 项目在之前的版本中都使用楷体作为图表标题，这是因为部分平台上中文字体字重不齐全，会导致生成的 pdf 无法显示中文粗体。

如果您的确需要修改，可以编辑 `sjtuthesis.cls` 的这两行：

```latex
\DeclareCaptionFont{sjtu@caption@font}{\zihao{5}\kaishu}
\DeclareCaptionFont{sjtu@subcaption@font}{\zihao{5}\normalfont}
```

改为：

```latex
\DeclareCaptionFont{sjtu@caption@font}{\zihao{5}\bf}
\DeclareCaptionFont{sjtu@subcaption@font}{\zihao{5}\bf}
```

如果没有效果，说明目前的字体集中文字体本身不包含粗体字重。请参考相关文档修改文档字体集 fontset `\documentclass[type=bachelor,fontset=xxxx]{sjtuthesis}`。https://github.com/sjtug/SJTUThesis/issues/308 中也有更为详细的解答可以参考。

**Reply by AlexaraWu** [reply]
目前 `develop` 分支本科模板已经修改为默认宋体加黑了，无需额外配置。

**Comment by skyzh** [comment]
目前 master 上的最新版已经默认宋体加粗了。
